### PR TITLE
docs: fix wrong type being displayed in docs

### PR DESCRIPTION
**Pull request by Megatank58**
**Please describe the changes this PR makes and why it should be merged:**
Interaction should be MessageComponentInteraction instead of ButtonInteraction

**Status and versioning classification:**

- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/interfaces/InteractionResponses.js [hunk 1]
```diff
@@ -11,14 +11,14 @@ const MessagePayload = require('../MessagePayload');
  */
 class InteractionResponses {
   /**
-   * Options for deferring the reply to an {@link Interaction}.
+   * Options for deferring the reply to a {@link MessageComponentInteraction}.
```
Comment (ImRodry) [inline src/structures/interfaces/InteractionResponses.js:14]: Why

Comment (Megatank58) [reply src/structures/interfaces/InteractionResponses.js:14]: Interaction can be both select menu interaction and button interaction but here only button interaction was mentioned

Comment (ImRodry) [reply src/structures/interfaces/InteractionResponses.js:14]: No, this mentioned Interaction and is still correct, please revert this

Comment (Megatank58) [reply src/structures/interfaces/InteractionResponses.js:14]: Oh my bad I'll close the pr

Comment (ImRodry) [reply src/structures/interfaces/InteractionResponses.js:14]: No, just revert this line alone

Comment (Megatank58) [reply src/structures/interfaces/InteractionResponses.js:14]: yeah sorry i misspelled

**Review by SpaceEEC — COMMENTED** [review]
I think this one should be updated too then:
https://github.com/discordjs/discord.js/blob/b43a441d4b3881f55a8c39a6773a7d3399a06dc1/src/structures/interfaces/InteractionResponses.js#L34

**Comment by Megatank58** [comment]
> I think this one should be updated too then:
> https://github.com/discordjs/discord.js/blob/b43a441d4b3881f55a8c39a6773a7d3399a06dc1/src/structures/interfaces/InteractionResponses.js#L34

Right I'll do that

**Review by SpaceEEC — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Pull request merged**